barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -123,18 +123,9 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
         return pows;
     }
 
-    static std::vector<FF> update_gate_challenges(const FF perturbator_challenge,
+    static std::vector<FF> update_gate_challenges(FF perturbator_challenge,
```

Comment:
const on first parameter should stay, no?

Reply:
No this doesn't mean anything and actually gives a linter warning. This is already copied into the function, the const here is a function-local annotation